shared_model/validators/validators_common.hpp

```diff
@@ -16,9 +16,13 @@ namespace shared_model {
      * A validator may read only specific fields.
      */
     struct ValidatorsConfig {
-      ValidatorsConfig(const uint64_t max_batch_size);
+      ValidatorsConfig(uint64_t max_batch_size,
+                       bool partial_ordered_batches_are_valid = false);
       /// Maximum allowed amount of transactions within a batch
       const uint64_t max_batch_size;
+
+      /// Turns on special batches validation logic for blocks from synchronizer
```

Comment:
```suggestion
      /// Batch meta can contain more hashes of batch transactions than it actually has. Used for block validation
```
So that there is nothing "special" 🤷‍♂️